src/utils/emlTemplate.j2

```diff
@@ -89,40 +92,65 @@
             </temporalCoverage>
         </coverage>
 
-    <maintenance>
-      <description>
-        <para/>
-      </description>
-      <maintenanceUpdateFrequency>asNeeded</maintenanceUpdateFrequency>
-    </maintenance>
 
+        <methods>
+            {% for method in methodList %}
+            <methodStep>
+                <description>
+                    <section>
+                        <title>{{ method.title.en }}</title>
+                        <para>{{ method.description.en }}</para>
+                    </section>
+                </description>
+            </methodStep>
+            {% endfor %}
+            <sampling>
```

Comment:
The sampling section seems to overlap with the following iso19115 concepts:
- https://wiki.esipfed.org/EX_Extent
- https://wiki.esipfed.org/EX_TemporalExtent
- https://wiki.esipfed.org/EX_GeographicDescription
- https://wiki.esipfed.org/MD_MaintenanceInformation